pkgs/by-name/lo/louvre/package.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  pkg-config,
+  fontconfig,
+  icu,
+  libdrm,
+  libGL,
+  libinput,
+  libX11,
+  libXcursor,
+  libxkbcommon,
+  mesa,
+  pixman,
+  seatd,
+  srm-cuarzo,
+  udev,
+  wayland,
+  xorgproto,
+}:
```

Comment:
this is quite nitpicky and since we should get a formatter soon feel free to ignore it but our style guide says that the `,` should be at the start of the line as described in https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#syntax `Function formal arguments are written as`